public/locales/fr/common.json

```
@@ -1,47 +1,47 @@
 {
   "scrobble": "Scrobble",
   "getScrobbling": "Début du scrobbling",
-  "loading": "Chargement...",
+  "loading": "Chargement ...",
```

```suggestion
  "loading": "Chargement...",
```

I'd rather keep this one as it was. Although it's not grammatically correct, keeping them together will avoid word-breaks from happenning at that space.

---

Added space to make it more "beautiful"